deploy/helm/flightctl/values.acm.yaml

```
@@ -10,3 +10,9 @@ kv:
   image:
     image: registry.redhat.io/rhel9/redis-7
     tag: 9.5-1736454843
+
+alertmanager:
+  image:
+    image: registry.redhat.io/rhacm2/prometheus-alertmanager-rhel9
+    tag: v2.13.2
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)